src/MinimumNodeService.cpp

```diff
@@ -82,6 +82,8 @@ void MinimumNodeService::initSetupFromNormal()
   // DEBUG_SERIAL << F("> reverting to Uninitialised mode") << endl;
   requestingNewNN = true;
   controller->sendMessageWithNN(OPC_NNREL);
+  controller->sendGRSP(OPC_MODE, getServiceID(), GRSP_OK);
```

Comment:
You already have a sendGRSP(OK) where this is called from on line 402, 445. 
I prefer to have the response messages sent in the part of the code where they are called from. 
If there is a problem with sequencing of response messages, then put the sendGRSP() before the call to initSetupFromNormal().

Reply:
OK. It makes sense.  Done.